docs/sandbox/apps/sarotate.md

```diff
@@ -18,10 +18,51 @@ sb install sandbox-sarotate
 
 ```
 
-### 2. URL
+### 2. Usage
+
+- After installing the role, add your SA path to the settings file in `/opt/sandbox/settings.yml`
+
+- Example
+
+  ``` { .yaml}
+  sarotate: 
+  remotes: ["google"]
+  sa_path: "/opt/sa/all" # Change this as needed
+  sleeptime: # Optional. Default is 300
+  rc_port: # Default is localhost:5572
+  rc_user: # Optional. Default is blank
+  rc_pass: # Optional. Default is blank
+  apprise: # Optional. Add an apprise url if it suits you
+
+  ```
+
+**remotes:** if you have a different union in your rclone.conf (typically located @ `/home/user/.config/rclone/`) you will want to supply that name here. The saltbox default is google.
```

Comment:
Clean up the wording. Not everyone has a union, but make it clear a union is OK and the role will resolve the backend remotes for building the config.